frontend/server/src/Email.php

```diff
@@ -38,6 +38,7 @@ public static function sendEmail(
 
         $mail = new \PHPMailer\PHPMailer\PHPMailer();
         $mail->IsSMTP();
+        $mail->SMTPDebug = SMTP::DEBUG_SERVER;
```

Comment:
```suggestion
        $mail->SMTPDebug = \PHPMailer\PHPMailer\SMTP::DEBUG_SERVER;
```
estoy adivinando, jaja. si no jala esto, puedes arreglarlo porfa?